open-reac/src/main/java/com/powsybl/openreac/ReactiveOpfNetworkUpdater.java

```diff
@@ -0,0 +1,50 @@
+/**
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.openreac;
+
+import com.powsybl.ampl.converter.AmplSubset;
+import com.powsybl.ampl.converter.DefaultAmplNetworkUpdater;
+import com.powsybl.commons.util.StringToIntMapper;
+import com.powsybl.iidm.network.*;
+
+/**
+ * @author Nicolas Pierre <nicolas.pierre at artelys.com>
+ */
+public class ReactiveOpfNetworkUpdater extends DefaultAmplNetworkUpdater {
```

Comment:
Is it on purpose here that we keep the name ReactiveOpf and not OpenReac?

Reply:
OpenReac is the good wording, I'll rename it.